src/main/java/org/lightcouch/PreconditionFailedException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2015 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package org.lightcouch;
+
+/**
+ * CouchDbException class for 412 precondition failed status codes
```

Comment:
is there a useful link to the couch docs we can give for this? I know that 412 is used for illegal DB names, anything else?